src/pages/patterns/common-actions/index.mdx

```diff
@@ -5,261 +5,252 @@ description: Common actions are frequently used actions that appear across diffe
-<AnchorLink>Copy</AnchorLink>
-<AnchorLink>Edit</AnchorLink>
-<AnchorLink>Next</AnchorLink>
-<AnchorLink>Refresh</AnchorLink>
+![Example of edit in an overflow menu](images/common-action-10.png)
 
-</AnchorLinks>
+</Column>
+</Row>
 
-### Add
+## Errors
 
-Add inserts an existing object to a list, set, or system. For example, adding a document to a folder.
+Errors occur when an action or process does not succeed. Error notifications can occupy full pages, form fields, notifications, and modals. Error notifications should provide context of what happened and a clear path to continue.
 
 <Row>
 <Column colLg={8}>
 
-![Example of add as a Button with an con in a Data Table](images/common-action-7.png)
+![Form example with inline notification](images/notification-usage-3.png)
 
 </Column>
 </Row>
 
-#### Hierarchy and placement
-
-Represent add with a primary button, button with an icon, `add--glyph` or `add--outline` icon.
+Consider redirecting the user to a previous state, a support page, or offering recommendations. Be honest and helpful.
 
-Depending on the relevance of the add action on the page, the emphasis can be high, medium or low. For example, a high emphasis action should use a single prominent button that makes it clear that other buttons have less importance in the hierarchy.
-
-#### Considerations
-
-- What are the implications of the add action for the user? Are there financial, access, or legal considerations?
-- A user may not have the correct permissions for this action
-- Inform the user if the action is permanent
-- What timeframe will the action take (seconds, minutes, hours, days)?
-- What should a user do if the addition fails?
-- Is this a single or bulk action?
+Some components, like text input and form field errors, are quite small and require more thoughtful approaches to the space and placement of error handling. Consider inline error notifications for these instances.
 
-### Copy
+## Next
 
-Copy creates a new identical instance of the selected object(s) in a distinct destination.
+Next advances the user to the next step in a sequence of steps, like in a wizard.
 
-Use the `copy` icon with a confirmation "copied" tooltip appearing post-click.
+Represent next with a [button with icon](/components/button/usage) or a standalone `forward` icon.
 
 <Row>
 <Column colLg={8}>
 
-![Example of copy in a code snippet](images/common-action-8.png)
+![Example of next as a button with icon](images/common-action-11.png)
 
 </Column>
 </Row>
 
-### Edit
+### Content guidelines
 
-Edit allows data or values to be changed. Edit commonly triggers a state change to the targeted object or input item.
+Be brief, honest, supportive, and human. Explain what happened and what can the user do to resolve the error.
 
-Offer edit as an option in a menu, or as a button or `edit` icon.
+For full-page and large modals, keep error messages no longer than three paragraph lines. For form errors, keep error messages no longer than two lines.
```

Comment:
I think the content guidelines are meant to go under errors

Reply:
Good catch, thank you!